app/routes/events/components/EventAdministrate/RegistrationTables.js

```diff
@@ -85,6 +91,63 @@ const getRegistrationInfo = (pool, registration) => {
   return registrationInfo;
 };
 
+const getConsentIcons = (LEGACY_photoConsent, photoConsents, eventSemester) => {
+  if (hasRegisteredConsent(photoConsents, eventSemester)) {
+    const { WEBSITE, SOCIAL_MEDIA } = PHOTO_CONSENT_DOMAINS;
+
+    const consentMessage = (photoConsent) =>
+      `Brukeren godkjenner ${
+        photoConsent?.isConsenting ? ' ' : 'IKKE '
+      } at bilder publiseres på ${
+        photoConsent?.domain === WEBSITE ? 'abakus.no' : 'sosiale medier'
+      }`;
+
+    const iconClass = (photoConsent) =>
+      cx(
+        photoConsent?.isConsenting ? styles.greenIcon : styles.redIcon,
+        photoConsent?.domain === WEBSITE
+          ? 'fa fa-desktop'
+          : 'fa fa-share-square'
+      );
```

Comment:
Move these functions outside of the outer function/component.

Reply:
✅